packages/storage/src/cookie-storage.spec.ts

```diff
@@ -0,0 +1,34 @@
+import CookieStorage from './cookie-storage';
+
+describe('CookieStorage', () => {
+    beforeEach(() => {
+        Object.defineProperty(window.document, 'cookie', {
+            writable: true,
+            value: `first_user=John;last_user=Ann;description=${encodeURIComponent(
+                'any small description =',
+            )}`,
+        });
+    });
+
+    it('retrieve an existing value by name', () => {
+        expect(CookieStorage.get('description')).toBe('any small description =');
+    });
+
+    it('returns null if there is no value by name', () => {
+        expect(CookieStorage.get('some')).toBeNull();
+    });
+
+    it('retrieve the existing value by name after setting', () => {
+        CookieStorage.set('agent', 'Smith');
+
+        expect(CookieStorage.get('agent')).toBe('Smith');
+    });
+
+    it('remove value by name', () => {
+        jest.spyOn(CookieStorage, 'set').mockImplementation();
+
+        CookieStorage.remove('last_user');
+
+        expect(CookieStorage.set).toHaveBeenCalledWith('last_user', '', { expires: new Date(0) });
```

Comment:
It might be great to check if the cookie with name `last_user` does not exist

Reply:
Are we testing all the conditions in `set` method?

Reply:
I am testing all conditions in set method. You can see this in the new commits